nifi-nar-bundles/nifi-prometheus-bundle/nifi-prometheus-reporting-task/src/main/java/org/apache/nifi/reporting/prometheus/api/PrometheusMetricsUtil.java

```diff
@@ -249,6 +257,18 @@
             .labelNames("instance")
             .register(JVM_REGISTRY);
 
+    private static final Gauge JVM_GC_RUNS = Gauge.build()
+            .name("nifi_jvm_gc_runs")
+            .help("NiFi JVM GC number of runs")
+            .labelNames("inctance", "gc_name")
```

Comment:
I think you meant `instance` for that label name.

Reply:
@MikeThomsen Thank you for finding it. My mistake 😝